lib/action_controller/serialization.rb

```diff
@@ -18,34 +19,25 @@ def serialization_scope
         respond_to?(_serialization_scope, true)
     end
 
+    # Deprecated
     def get_serializer(resource)
-      @_serializer ||= @_serializer_opts.delete(:serializer)
-      @_serializer ||= ActiveModel::Serializer.serializer_for(resource)
-
-      if @_serializer_opts.key?(:each_serializer)
-        @_serializer_opts[:serializer] = @_serializer_opts.delete(:each_serializer)
-      end
-
-      @_serializer
+      fail ActiveModel::Serializer::Error, "ActionController::Serialization##{__method__} has been removed. "\
+        "Please use ActiveSupport::Serializer#build instead"
     end
```

Comment:
Glad you support that. I'm really unhappy with straight-up removing them, but didn't come up with any code solutions that I thought were an improvement over just failing.

Although, I think my implementation is wrong, in that if someone overrides these methods, they won't get a failure.  So, that leaves me with doing a method_defined? check at boot/inheritance and/or taking a look at method missing.. ugh.. I'm going to take another look to see if I can keep the code alive a little longer.